script.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line no-unused-vars
+function clickToSwitch() {
+  const switchD = document.getElementById('switch-display');
```

Comment:
- [ ] great job so far. Everything looks fine. Kindly reposition your close icon like that of Figma
- [ ] please add the size of the close icon, so it is more visible
- [ ] please add the font size and weight(600) of the menu links.

yours
![Screenshot (423)_LI](https://user-images.githubusercontent.com/65298048/188514199-d85e96d9-4a81-46c3-8a66-ce600350eeea.jpg)
figma
![Screenshot (421)](https://user-images.githubusercontent.com/65298048/188514231-6b1f5b1c-0f37-4605-bde5-6087080f6cbd.png)

Reply:
Completed